packages/devtools-network-console/src/ui/Authorization/index.tsx

```diff
@@ -3,50 +3,70 @@
 
 import * as React from 'react';
 import { useDispatch } from 'react-redux';
-import { Pivot, PivotItem, Stack, MessageBar, MessageBarType } from '@fluentui/react';
+import { Stack, MessageBar, MessageBarType } from '@fluentui/react';
 import { INetConsoleAuthorization, NetworkConsoleAuthorizationScheme } from 'network-console-shared';
 
 import BasicAuthorization from './BasicAuthorization';
 import BearerToken from './BearerToken';
 import { makeSetAuthorizationSchemeAction } from 'actions/request/auth';
 import { IEnvironmentAuthorizationState } from 'store';
-import { THEME_OVERRIDE } from 'themes/vscode-theme';
+import { Radio } from '@microsoft/fast-components-react-msft';
+import { HideUnless } from 'ui/generic/HideIf';
+import CommonStyles from '../common-styles';
 
 export interface IAuthorizationProps {
     requestId: string;
     authorization: INetConsoleAuthorization;
     environmentAuth?: IEnvironmentAuthorizationState;
 }
 
-const TYPE_TO_INDEX = new Map<NetworkConsoleAuthorizationScheme, number>([
-    ['inherit', 0],
-    ['none', 1],
-    ['token', 2],
-    ['basic', 3],
-]);
 export default function Authorization(props: IAuthorizationProps) {
     const token = props.authorization.token;
     const basic = props.authorization.basic;
-    let index = TYPE_TO_INDEX.get(props.authorization.type) || 0;
     const env = props.environmentAuth;
     const dispatch = useDispatch();
 
     return (
-        <Pivot
-            initialSelectedIndex={index}
-            onLinkClick={(pivotItem) => {
-                if (pivotItem) {
-                    // TODO: Improve this hack
-                    const key = ((pivotItem as any).key as string).substr(2) as NetworkConsoleAuthorizationScheme;
-                    dispatch(makeSetAuthorizationSchemeAction(props.requestId, key));
-                }
-            }}
-            styles={{
-                root: THEME_OVERRIDE.smallPivotRoot,
-                link: THEME_OVERRIDE.smallPivotButtons,
-                linkIsSelected: THEME_OVERRIDE.smallPivotButtons,
-            }}>
-            <PivotItem headerText="Inherited" key="inherit">
+        <>
```

Comment:
im not familiar with this structure, what is the function of <> ?

Reply:
In React, where components are expected to return a single child, it's (for historical reasons) invalid markup to return multiple items, even though it's semantically legal. To work around this, React 16.2 introduced [fragments](https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html). This is that.